tests/check_codeblocks.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Script that checks if the Code::Blocks project is complete.
+# TODO: If the project is incomplete generate a patch file to make it complete.
+
+# Determine path of the current script, and go to the ES project root.
```

Comment:
Add relevant `set` here, e.g. pipefail and others.

Reply:
`-u` certainly, and `-e` if the script logic supports it.

Pipefail would be added mostly to avoid it being forgotten if the script is refactored to use pipelines.

(If you'd prefer to write this script in Python / nodejs that's totally ok too.)